xml/System/MemoryExtensions.xml

```diff
@@ -61,7 +61,15 @@
         <param name="text">The target string.</param>
         <summary>Creates a new <see langword="ReadOnlyMemory&lt;Char&gt;" /> over the portion of the target string.</summary>
         <returns>The read-only character memory representation of the string, or <see langword="default" /> if <paramref name="text" /> is <see langword="null" />.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Returns default when `text` is `null`.
+
+          ]]></format>
+        </remarks>
```

Comment:
this information is already present  in the `<returns>` tag:

> `The read-only character memory representation of the string, or <see langword="default" /> if <paramref name="text" /> is <see langword="null" />`.

Should we keep it in both places?

BTW this particular example could be simplified (cc @carlossanlop):

```suggestion
        <remarks>Returns default when <paramref name="text" /> is <see langword="null" />.</remarks>
```